src/openrct2/rct1/RCT1.h

```diff
@@ -306,8 +306,15 @@ struct rct1_vehicle : RCT12SpriteBase
     uint8_t var_C4;
     uint8_t animation_frame;
     uint8_t pad_C6[0x2];
-    uint16_t var_C8;
-    uint16_t var_CA;
+    union
+    {
+        uint32_t additional_animation_var;
+        struct
+        {
+            uint16_t var_C8;
+            uint16_t var_CA;
```

Comment:
Do we still need this variable? It looks like it was only ever used a temporary variable.